### DISCUSSION TITLE
Which Tags are available / supportet in actual version?

### DISCUSSION THREAD

**Discussion by Timbuktu1982** [EnterTheNameHere/esdoc-monorepo#4] [Q&A] [answered]
Hi,

is there an overview of all tags or could one be kept here in the repo in the future?

THX

**Comment by EnterTheNameHere** [comment, marked as answer]
I've added list of tags to [README.md](https://github.com/EnterTheNameHere/esdoc-monorepo#tags) for now. They link to TAGS.md for their respective examples.

If you miss some tag, let me know.

I also have to make initiation of new project config easier and user friendly. It's more about replacing the old esdoc with updated version where you already have existing esdoc config without the need of making changes.

**Comment by Timbuktu1982** [comment]
THX
